integration_tests/CMakeLists.txt

```diff
@@ -675,12 +675,12 @@ RUN(NAME structs_32          LABELS cpython llvm c)
 RUN(NAME structs_33          LABELS cpython llvm c)
 RUN(NAME structs_34          LABELS cpython llvm c)
 
-RUN(NAME symbolics_01        LABELS cpython_sym c_sym)
-RUN(NAME symbolics_02        LABELS cpython_sym c_sym)
-RUN(NAME symbolics_03        LABELS cpython_sym c_sym)
-RUN(NAME symbolics_04        LABELS cpython_sym c_sym)
-RUN(NAME symbolics_05        LABELS cpython_sym c_sym)
-RUN(NAME symbolics_06        LABELS cpython_sym c_sym)
+# RUN(NAME symbolics_01        LABELS cpython_sym c_sym)
```

Comment:
All the commented-out programs fail for the `assert` statement, right?

Reply:
yes correct . Like talking about `symbolics_01.py`
```
from sympy import Symbol, pi
from lpython import S

def main0():
    x: S = Symbol('x')
    y: S = Symbol('y')
    x = pi
    z: S = x + y
    print(z)
    assert(z == pi + y)
    assert(z != S(2)*pi + y)

main0()
```
Everything other than the assert lines would work properly !